test/HubPool.Fees.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from "chai";
```

Comment:
please work through this test file! the second test covers a full commented evaluation of fees over a period of time.